book.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'rental'
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] In this file, an `add_rental` method too needs to be added as required and inside it, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

Reply:
* As mentioned by the previous CR this issue is still pending Kindly fix it [refer to this lesson link](
https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)